installer/conf/omsagent.d/squid.conf

```diff
@@ -0,0 +1,72 @@
+# Linux Squid Proxy Log Monitoring Solution for Operations Management Suite
+# Developed by Alessandro Cardoso
+# 
+# Important: This scrpt assumes that the squid.conf logformat is configured by default as per below. 
```

Comment:
fixed. Unit Tests added at the end of the file. @sugr4 please review and approve